Makefile

```diff
@@ -308,11 +313,13 @@ docker/build:
 docker/run: DOCKER_RUN_FLAGS ?= --rm
 docker/run: AWS_DEFAULT_REGION ?= us-east-1
 docker/run: target ?= help
+docker/run: | guard/env/TARDIGRADE_CI_PATH guard/env/TARDIGRADE_CI_PROJECT
```

Comment:
I don't understand why 'guard/env/TARDIGRADE_CI_PATH' is needed here.

Reply:
The target is explicitly using the `TARDIGRADE_CI_PATH` env, so this prevents errors if the user mistakenly passes `TARDIGRADE_CI_PATH =` (i.e. either an unset value or an empty string).

Reply:
It seems a bit unlikely, but it doesn't hurt to check.